tasks/website_formatter/src/snapshots/cli.snap

```diff
@@ -13,10 +13,12 @@ search: false
 ## Mode Options:
 - **`    --init`** &mdash; 
   Initialize `.oxfmtrc.json` with default values
-- **`    --lsp`** &mdash; 
-  Start language server protocol (LSP) server
 - **`    --migrate`**=_`SOURCE`_ &mdash; 
   Migrate configuration to `.oxfmtrc.json` from specified source Available sources: prettier
+- **`    --lsp`** &mdash; 
+  Start language server protocol (LSP) server
+- **`    --stdin-filepath`**=_`PATH`_ &mdash; 
+  Read from stdin and use this filepath to determine parser Formatted code is written to stdout
```

Comment:
The grammar of this sentence is incorrect, can you copy what's written here https://prettier.io/docs/options#file-path

> Specify the file name to use to infer which parser to use.